admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/component/navigation_bar/VerticalNavigationBarItem.kt

```diff
@@ -0,0 +1,63 @@
+package net.thechance.mena.admin_panel.presentation.component.navigation_bar
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+
+@Composable
+fun VerticalNavigationBarItem(
+    isSelected: Boolean,
+    unselectedIcon: Painter,
+    selectedIcon: Painter,
+    title: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val painter = if (isSelected) selectedIcon else unselectedIcon
+    val interactionSource = remember { MutableInteractionSource() }
+    val textColor = if (isSelected) Theme.colorScheme.brand.brand
+    else Theme.colorScheme.shadeSecondary
```

Comment:
you need format this
 
```suggestion
    val textColor = if (isSelected) Theme.colorScheme.brand.brand else Theme.colorScheme.shadeSecondary
```